src/js/components/Layer/LayerContainer.js

```diff
@@ -46,6 +49,12 @@ const LayerContainer = forwardRef(
     const anchorRef = useRef();
     const containerRef = useRef();
     const layerRef = useRef();
+    const portalContext = useContext(PortalContext) || defaultPortalContext;
+    const portalId = useMemo(() => portalContext.length, [portalContext]);
+    const nextPortalContext = useMemo(() => [...portalContext, portalId], [
```

Comment:
Can we use the same PortalContext and reference both for Drop and Layer.

Reply:
They can share. Docs here: https://reactjs.org/docs/context.html#contextprovider

"Providers can be nested to override values deeper within the tree.

All consumers that are descendants of a Provider will re-render whenever the Provider’s value prop changes."